spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/openapi/dto/Message.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.openapi.dto;
+
+public class Message {
+
+    private String role;
+    private String content;
+
+    Message(String role, String content) {
+        this.role = role;
+        this.content = content;
+    }
+
+    Message() {
```

Comment:
Do you need an empty constructor ?

Reply:
it was required for deserializing the response